lib/facilities/multi_client.rb

```diff
@@ -15,7 +16,7 @@ def run(requests)
       requests.map do |req|
         if req.response.timed_out?
           Rails.logger.error "GIS request timed out: #{req.url}"
-          raise Common::Client::Errors::RequestTimeout
+          raise Common::Exceptions::ClientError.new('Facility request timed out', {})
```

Comment:
You can look at the Exception class for Validations, that one attempts to build an array, but I think for this variety of error you might need a more generic version of that exception class, perhaps if you create one then the Validation one could just inherit from it. I'm happy to elaborate via slack.